ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -21,6 +27,9 @@ Install RHODS
           FAIL    Provided test envrioment is not supported
       END
   END
+  IF  "${UPDATE_CHANNEL}" != "stable" and "${UPDATE_CHANNEL}" != "beta"
```

Comment:
This should be OR not AND

Reply:
My understanding is that it should be `and`, since we want to create the CR only if the channel is neither stable nor beta (i.e. alpha or fast).
If I use `or` it would create the DSC CR also when the channel is `stable` (false or true == true) or when it's `beta` (true or false == true), and of course when it's `alpha`/`fast` (true or true == true). With the `and` as it is now it would be (false and true == false) (true and false == false) (true and true == true).